rpm_spec/qubes-desktop-linux-manager.spec

```diff
@@ -77,6 +77,8 @@ ln -sf . %{name}-%{version}
 mkdir -p $RPM_BUILD_ROOT/etc/xdg/autostart
 cp -r autostart/qui-domains.desktop $RPM_BUILD_ROOT/etc/xdg/autostart
 cp -r autostart/qui-devices.desktop $RPM_BUILD_ROOT/etc/xdg/autostart
+mkdir -p $RPM_BUILD_ROOT/usr/share/icons/Adwaita/22x22/devices/
+cp -r icons/22x22/generic-usb.png $RPM_BUILD_ROOT/usr/share/icons/Adwaita/22x22/devices/generic-usb.png
```

Comment:
This will not work if user change icon theme. But since this is the only one installed (besides `hicolor`), I think it's ok for now.
Some references:
https://specifications.freedesktop.org/icon-theme-spec/icon-theme-spec-latest.html
https://portland.freedesktop.org/doc/xdg-icon-resource.html